playbooks/tasks/boot_instance_attempt.yml

```diff
@@ -0,0 +1,104 @@
+---
+# All tasks in this file are skipped if boot_success is true.
+# This is to avoid creating duplicate sets of instances
+# once a set has booted.
+
+# Tasks in this file must not fail, otherwise the loop will end
+# without retries.
+
+# attempt_success is used to determine if the current attempt succeeded
+
+- name: Provision a set of public cloud instances
+  rax:
+    name: "{{ name }}"
+    flavor: "{{ flavor }}"
+    image: "{{ image }}"
+    count: "{{ count }}"
+    key_name: "{{ keyname }}"
+    region: "{{ item }}"
+    wait: yes
+    wait_timeout: 900
+    auto_increment: no
+  delegate_to: localhost
```

Comment:
I'd prefer if we had a feature to build from a random region from a list of regions , instead of building this.